src/transport/SessionManager.h

```diff
@@ -251,6 +251,28 @@ class DLL_EXPORT SessionManager : public TransportMgrDelegate
     using SessionHandleCallback = bool (*)(void * context, SessionHandle & sessionHandle);
     CHIP_ERROR ForEachSessionHandle(void * context, SessionHandleCallback callback);
 
+    //// FabricTable::Delegate Implementation ////
+    void OnFabricDeletedFromStorage(FabricTable & fabricTable, FabricIndex fabricIndex) override
+    {
+        (void) fabricTable;
+        this->FabricRemoved(fabricIndex);
```

Comment:
This is going to call `FabricRemoved` twice in the controller case, no?  Shouldn't we remove the existing call to `FabricRemoved` there like the server one was removed?  See `ControllerFabricDelegate`.

Reply:
Good catch